master/internal/core.go

```diff
@@ -772,7 +772,7 @@ func (m *Master) Run(ctx context.Context) error {
 
 	gzipConfig := middleware.GzipConfig{
 		Skipper: func(c echo.Context) bool {
-			return !gzipSkipPaths[c.Path()]
+			return gzipSkipPaths[c.Path()]
```

Comment:
hah. if that was working just fine "inverted", maybe there should not be any "skip" logic at all and it can just work "globally".

Reply:
I agree. I will test removing the skipping logic and if it does not seem to have any negative effects, which I am guessing will be the case, then I will update the PR.

Reply:
Perhaps revisiting how https://github.com/determined-ai/determined/pull/4517/files got tested would be helpful

Reply:
Updated this PR after some testing and findings: https://github.com/determined-ai/determined/pull/5831#issuecomment-1405599115

Reply:
I'm not sure if this is what Ilia means but wdy think about whitelisting the paths that we do want gzip compressed if they fall neatly under a small set of directories instead of the other way? this could lead to fewer surprises.

Reply:
I will look into this. All of this logic is already constrained to the `/det` WebUI base route, but maybe there are some more specific directories that we can filter for. I will look into this now.

Reply:
oh I didn't notice these were scoped to /det. are we gzip doc assets as well? do we want that

Reply:
update: I don't immediately see where this is constrained to just `/det` maybe if we look into `webuiGroup.Use(GZIPMIDDLEWARE)` it'd do that?

Reply:
@hamidzr you are right. I mistakenly thought this logic was only being run in the `webuiGroup` defined in `core.go` but that was not the case. I have updated the logic now.